.gitignore

```diff
@@ -7,10 +7,12 @@
 .settings
 
 # Build folder and vendor folder are generated code; no need to version this
-build/*
-vendor/*
-temp/*
-composer.phar
+build/
+vendor/
+temp/
+tools/
+*.phar
 
 # By default the phpunit.xml.dist is provided; you can override this using a local config file
 phpunit.xml
+phive.xml
```

Comment:
phive.xml is like composer.lock for composer. We might consider the commit this to lock our dependencies.